packages/commons/src/config/selfcareConfig.ts

```diff
@@ -10,4 +10,16 @@ export const SelfCareConfig = z
     selfcareApiKey: c.SELFCARE_V2_API_KEY,
   }));
 
-export type SelfCareConfig = z.infer<typeof SelfCareConfig>;
+export type SelfCareClientConfig = z.infer<typeof SelfCareClientConfig>;
+
+export const SelfcareConsumerConfig = z
+  .object({
+    SELFCARE_USERS_TOPIC: z.string(),
```

Comment:
since the other selfcare consumer uses a different topic (sc-contracts) if we want a common config I recommend a more generic name
 ```suggestion
    SELFCARE_TOPIC: z.string(),
```